doc_src/auto_examples/index.rst

```diff
@@ -0,0 +1,125 @@
+:orphan:
+
+
+
+.. _sphx_glr_auto_examples:
+
+This is the Franck lab gallery
+==============================
+
+Below is a gallery of examples using our 
+various functions.
```

Comment:
@guinneak198 : did you create this file, or was it auto-generated?

Reply:
I created it

Reply:
you typed out this whole file by hand?

Reply:
Obsolete now --  I fixed this